ext/opentelemetry-ext-jaeger/examples/jaeger_exporter_example.py

```diff
@@ -2,11 +2,11 @@
 
 from opentelemetry import trace
 from opentelemetry.ext import jaeger
-from opentelemetry.sdk.trace import Tracer
+from opentelemetry.sdk.trace import TracerSource
 from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
 
-trace.set_preferred_tracer_implementation(lambda T: Tracer())
-tracer = trace.tracer()
```

Comment:
So it seems there isn't any unit test for this? But even the linter should have catched this. I guess we should update tox.ini to lint that.